float32 accX
float32 accY
float32 accZ
float32 gyroX
float32 gyroY
float32 gyroZ